spring-shell-autoconfigure/src/main/java/org/springframework/shell/boot/LineReaderAutoConfiguration.java

```diff
@@ -45,15 +45,15 @@ public class LineReaderAutoConfiguration {
 
 	private Parser parser;
 
-	private CommandRegistry commandRegistry;
+	private CommandCatalog commandRegistry;
```

Comment:
I'll mention this once  here as its a **[NIT]** but worth pointing out... the var name needs to change as well. Its still reads "Registry" below in the code.

Reply:
ack